litmus-portal/frontend/src/pages/MyHub/customMyHubCard.tsx

```diff
@@ -141,8 +141,8 @@ const CustomMyHubCard: React.FC<customMyHubCardProp> = ({
           src={`./icons/${
             hub.HubName === 'Chaos Hub'
               ? 'myhub-litmus.svg'
-              : hub.HubName === 'Kubera Chaos Hub'
-              ? 'kubera-chaos-hub.svg'
+              : hub.HubName === 'Litmus Chaos Hub'
```

Comment:
@ishangupta-ds  This condition won't be needed. Only two logos are needed one for Chaos hub and one default one.
cc: @amityt 
Only one condition should do the work like below.
```suggestion
              hub.HubName === 'Chaos Hub'
              ? 'myhub-litmus.svg': 'mu-hub-charts.svg'

```